packages/ia-components/components/bookreader-component/bookreader-wrapper-main.jsx

```diff
@@ -110,19 +115,18 @@ export default class BookReaderWrapper extends Component {
   }
 
   render() {
-    const { userSignedIn, item, baseHost } = this.props;
-    const base64item = btoa(item);
+    const { userSignedIn, baseHost } = this.props;
     return (
       <section className="bookreader-wrapper liner-notes" {...this.props}>
-        <item-navigator
-          baseHost={baseHost}
-          itemtype="bookreader"
-          signedIn={userSignedIn}
-          item={base64item}
-          class="focus-on-child-only"
+        <ia-bookreader
+          signedin={userSignedIn}
```

Comment:
react treats these as `unknown attributes` - since `ia-bookreader` is a special element, react recommends them to be lowercase.. [doc](https://reactjs.org/blog/2017/09/08/dom-attributes-in-react-16.html)